### DISCUSSION TITLE
Maintenance and development

### DISCUSSION THREAD

**Discussion by 0x19** [0x19/goesl#35] [Announcements]
Hello everyone!

I am sorry for not maintaining codebase of this project, I just don't have capacity at this moment to deal with it.

Whole project, if you ask me is more to show how to interact with FS and is not intended at this moment to be used in wide production.

I may be able later on, not sure when, jump in and start the rewrite to make it more resilient, robust and stable. 

Until then, if anyone wants to start developing/maintaining, please reply here under the discussion.

All the best,
Nevio

**Comment by yasarluo** [comment]
Hi, i want to maintain this project, how can i join it.

**Reply by 0x19** [reply]
Hey. Sorry for slow reply. In order to start maintaining this project, I believe the best thing would be to start with the PRs and additionally, whenever someone wants to merge anything, to use @0x19  tag so I am notified. Additional way how it can be done is by adding me to telegram. My telegram username is @nevionix. There you can shout on me whenever you need PR review etc... 

Once I am convinced someone has passion to take over the project, will be added to repository.
